packages/itwin/reports-config-widget/src/ReportsConfigWidget.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import type { Localization } from "@itwin/core-common";
+import { IModelApp } from "@itwin/core-frontend";
+
+export class ReportsConfigWidget {
+  private static _isInitialized = false;
+  private static _i18nNamespace: any;
```

Comment:
why is this typed as any when its just a string? also why is this property being lazy initialized?

Reply:
Oops missed it.